cartographer_ros/cartographer_ros/map_builder_bridge.cc

```diff
@@ -259,12 +314,21 @@ visualization_msgs::MarkerArray MapBuilderBridge::GetTrajectoryNodeList() {
       const ::geometry_msgs::Point node_point =
           ToGeometryMsgPoint(node_id_data.data.global_pose.translation());
       marker.points.push_back(node_point);
+      if (node_id_data.id.node_index == num_inter_submap_constrained_nodes) {
+        marker_alpha = 0.2;
```

Comment:
perhaps <= may be better for readability? (also consider using the default above if that makes it easier)